multiplex.py

```diff
@@ -14,28 +14,34 @@ class t_global(object):
 
 
 def process_options():
-    parser = argparse.ArgumentParser(description = 'Translate a JSON with multi-value parameters into a JSON with single-value parameters')
+    """Process arguments from command line"""
+    parser = argparse.ArgumentParser(
+        description = 'Translate a JSON with multi-value parameters into a
+                       JSON with single-value parameters')
 
     parser.add_argument('--input',
                         dest = 'input',
                         help = 'JSON file with multi-value parameters',
-                        default = 'bench-params.json',
+                        default = 'mv-params.json',
                         type = str)
 
     t_global.args = parser.parse_args()
     return(0)
 
 
 def dump_json(obj, format = 'readable'):
+    """Dump json in readable or parseable format"""
+    # Parseable format has no indentation
+    indentation = None
     if format == 'readable':
-        return json.dumps(obj, indent = 4, separators = (',', ': '), sort_keys = True)
-    elif format == 'parsable':
-        return json.dump(obj, separators = (',', ':'), sort_keys = True)
+        intentation = 4
 
-    return(None)
+    return json.dumps(obj, indent = indentation, separators = (',', ': '),
```

Comment:
Small nit here...this change forces separators to have `': '` for both readable and parsable...one of the intents of parsable (besides being easy to parse obviously) was to conserve bytes...adding extra spaces can actually add up a lot on large json objects.

Reply:
Please see the solution provided in https://github.com/perftool-incubator/multiplex/pull/16/commits/6aa6867060b9496751725c025b591a97535d4136